helm/prometheus-rules/templates/kaas/tenet/alerting-rules/cluster-autoscaler.rules.yml

```diff
@@ -26,13 +26,14 @@ spec:
         topic: cluster-autoscaler
     - alert: ClusterAutoscalerFailedScaling
       annotations:
-        description: '{{`Cluster-Autoscaler on {{ $labels.cluster_id }} has failed scaling up.`}}'
+        description: '{{`Cluster-Autoscaler on {{ $labels.cluster_id }} has failed scaling up {{ $value | printf "%.0f" }} times recently.`}}'
         runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/cluster-autoscaler-scaling/
-      expr: cluster_autoscaler_failed_scale_ups_total{provider=~"capa|capz|eks"} > 0
-      for: 15m
+      expr: increase(cluster_autoscaler_failed_scale_ups_total{provider=~"capa|capz|eks"}[15m]) > 3 and rate(cluster_autoscaler_failed_scale_ups_total{provider=~"capa|capz|eks"}[5m]) > 0
```

Comment:
These values aren't right I don't think. We had a legit issue that this alert should have caught with `cicdprod` on the 27th but this doesn't catch it.

Reply:
Ok thanks I'll investigate

Reply:
<img width="2051" alt="image" src="https://github.com/user-attachments/assets/109043ad-5ef2-41b3-a197-3f093b60cfcd" />
It might be changing it to  > 1 would fire it properly WDYT?

Reply:
Yeah removed `for`, dropped the provider and adjusted the increase